### ISSUE TITLE
`InlineStack` component fails to render without express import using `Pagination`

### ISSUE THREAD

**Issue by dotunwrap** [ownego/polaris-vue#417]
As a developer using the [`ResourceList`](https://github.com/ownego/polaris-vue/blob/b6bdc37cb273ac87db2ee2b7085cb46ae68fbacd/src/components/ResourceList/ResourceList.vue) component with the `pagination` prop set, Vue fails to render the [`InlineStack`](https://github.com/ownego/polaris-vue/blob/b6bdc37cb273ac87db2ee2b7085cb46ae68fbacd/src/components/InlineStack/InlineStack.vue) component due to it not being expressly imported by the [`Pagination`](https://github.com/ownego/polaris-vue/blob/b6bdc37cb273ac87db2ee2b7085cb46ae68fbacd/src/components/Pagination/Pagination.vue) component.

This occurs when not registering the library as a plugin as the components are not globally available.

**Comment by blacklen** [comment]
Hi @dotunwrap, could you please provide more context, such as the code you are using, so I can better understand this issue?

**Comment by dotunwrap** [comment]
@blacklen I cannot provide the code I am specifically using as it is for work; however, I can provide a similar example.

```vue
<script setup>
  import { ResourceList, ResourceItem } from '@ownego/polaris-vue';

  const pagination = {
    hasNext: true,
    onNext: () => {},
  };

  const testData = ref({id: 1, test: 'test'});
</script>

<template>
  <ResourceList
    :pagination=pagination <!-- Implicitly uses the Pagination component when this prop is used -->
    :resource-name="{ singular: 'test', plural: 'tests' }"
    :items="testData"
  >
    <template v-for="test in testData" :key="test.id">
      <ResourceItem
        :id=test.id
        :name="test.test"
      >
        <!-- Item markup here -->
      </ResourceItem>
    </template>
  </ResourceList>
</template>
```

This code will attempt to render pagination controls on the resource list due to the `pagination` prop being provided to the component. In `ResourceList`, this will inherently render the `Pagination` component to accomplish that. The `Pagination` component utilizes the component `InlineStack`; however, `Pagination` does not directly import `InlineStack` itself. This means that using this library without registering it as a plugin in my `App.vue` file (I prefer not to for build size reasons) leaves me with a Vue warning in the console regarding a component that's unable to render (`InlineStack` due to it not being imported and available). The solution here is simply to import `InlineStack` in the `Pagination` component as it seemingly should be in the first place.

**Comment by blacklen** [comment]
Fixed on `v2.1.19`
